frontend/src/queries/schema/schema-general.ts

```diff
@@ -2819,8 +2819,6 @@ export interface FunnelsActorsQuery extends InsightActorsQueryBase {
     /** Index of the step for which we want to get the timestamp for, per person.
      * Positive for converted persons, negative for dropped of persons. */
     funnelStep?: integer
-    /** Custom step numbers to get persons for. This overrides `funnelStep`. Primarily for correlation use. */
-    funnelCustomSteps?: integer[]
```

Comment:
is it safe to remove this field, would this have been set for any user? if yes then we'll get pydantic errors while reading existing queries